math-libs/BLAS/CMakeLists.txt

```
@@ -83,9 +83,12 @@ list(APPEND _blas_subproject_names hipBLASLt)
 # rocBLAS
 ##############################################################################
 
-set(rocBLAS_optional_deps)
+set(_rocBLAS_optional_runtime_deps)
```

```suggestion
set(rocBLAS_optional_runtime_deps)
```
Naming would inconsistent otherwise, but we could harmonize this across the project. Maybe in a follow-up?